packages/app/src/explorer/Explorer.tsx

```diff
@@ -22,6 +23,9 @@ function Explorer(props: Props) {
     btnRef.current?.focus();
   }, [btnRef]);
 
+  useKeyboardEvent('Home', focusFirst, [], { target: ref });
+  useKeyboardEvent('End', focusLast, [], { target: ref });
```

Comment:
Why not just put those in the buttons' event handler? It would keep everything together like before, and the event handler is already registered, so it'd just be adding a couple of `case`s to the `switch`.